### PR TITLE
Initial inclusion of eopkg4-bin

### PR DESCRIPTION
**Pull request by sheepman4267**
**Summary**
Includes eopkg4-bin, which is eopkg4 compiled with nuitka. The resulting package doesn't depend on python, which whould make it much more resilient to system python upgrades.

**Test Plan**

- Build `eopkg4-bin` from this PR
- Install it
- Check to see that all eopkg functions work correctly using the command `eopkg4-bin`. The same caveats apply as with `eopkg4`.

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/e/eopkg4-bin/package.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name       : eopkg4-bin
+version    : 4.0.0
+release    : 1
+source     :
+    - git|https://github.com/getsolus/eopkg : 38e98ba789d124bec649141ee569bb472832d813
+homepage   : https://github.com/getsolus/eopkg
+license    : GPL-2.0-or-later
+component  : system.base
+summary    : "Python3 port of eopkg/pisi. WARNING: For testing only."
```
Comment (davidjharder) [inline packages/e/eopkg4-bin/package.yml:9]: quotes again

Comment (sheepman4267) [reply packages/e/eopkg4-bin/package.yml:9]: This one has quotes on purpose. The : in the summary freaks out YAML.

Comment (davidjharder) [reply packages/e/eopkg4-bin/package.yml:9]: as you were then

**Review by ermo — APPROVED** [review]
Let's get this in. I've tested it with both packagekit backends and it works well enough to be let into the repo now IMO.

**Pull request merged**